todo/views.py

```diff
@@ -95,7 +95,6 @@ def update_task(request, **kwargs):
 def delete_task(request, **kwargs):
     task_id = kwargs.get("task_id")
     task = Task.objects.get(pk=task_id)
-    task.delete()
```

Comment:
Please add task.delete() I think you removed it, this makes sure that the tasks is actually deleted, if you do this. The view will pass but the object instance of Todo will not be deleted in the database.

Make sure you read Django docs for real views usage. Also use Class Based views.